apps/src/storage/dataBrowser/dataVisualizer/ChartPlaceholder.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import color from '../../../util/color';
+
+const styles = {
+  container: {position: 'relative', textAlign: 'center'},
+  text: {
+    position: 'absolute',
+    width: '100%',
+    bottom: '50%',
+    fontFamily: '"Gotham 5r", sans-serif, sans-serif',
+    fontSize: 20,
+    color: color.dark_charcoal
+  }
+};
+
+class ChartPlaceholder extends React.Component {
+  render() {
+    return (
+      <div style={styles.container}>
```

Comment:
If this is the whole class, and it's only used from one file, I wonder about making it its own class in the file that calls it, rather than it's own file.